operator/helper/multiline_test.go

```diff
@@ -420,6 +421,90 @@ func TestNewlineSplitFunc(t *testing.T) {
 	}
 }
 
+func TestNoSplitFunc(t *testing.T) {
+	testCases := []tokenizerTestCase{
+		{
+			Name: "OneLogSimple",
+			Raw:  []byte("my log\n"),
+			ExpectedTokenized: []string{
+				"my log\n",
+			},
+		},
+		{
+			Name: "TwoLogsSimple",
+			Raw:  []byte("log1\nlog2\n"),
+			ExpectedTokenized: []string{
+				"log1\nlog2\n",
+			},
+		},
+		{
+			Name: "TwoLogsCarriageReturn",
+			Raw:  []byte("log1\r\nlog2\r\n"),
+			ExpectedTokenized: []string{
+				"log1\r\nlog2\r\n",
+			},
+		},
+		{
+			Name:              "NoTailingNewline",
+			Raw:               []byte(`foo`),
+			ExpectedTokenized: []string{"foo"},
+		},
+		{
+			Name: "HugeLog100",
+			Raw: func() []byte {
+				return generatedByteSliceOfLength(100)
+			}(),
+			ExpectedTokenized: []string{
+				string(generatedByteSliceOfLength(100)),
+			},
+		},
+		{
+			Name: "HugeLog300",
+			Raw: func() []byte {
+				return generatedByteSliceOfLength(300)
+			}(),
+			ExpectedTokenized: []string{
+				"abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuv",
+				"wxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqr",
+				"stuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmn",
+			},
+		},
+		{
+			Name: "EOFBeforeMaxLogSize",
+			Raw: func() []byte {
+				return generatedByteSliceOfLength(350)
+			}(),
+			ExpectedTokenized: []string{
+				"abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuv",
+				"wxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqr",
+				"stuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmn",
+				"opqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijkl",
+			},
+		},
+	}
+
+	for _, tc := range testCases {
+		splitFunc := SplitNone(100)
```

Comment:
I would make this a constant and tie tests to it. Equal to, slightly greater, much greater, etc.

Reply:
added more tests

Reply:
This didn't really address what I was pointing out. The value `100` on this line should be a constant, because several of the tests depend on it. If we make it a constant, we can make the tests depend on the constant, which can then change without breaking tests.

Reply:
added constant.